### PR TITLE
Make branch parameter optional in 'pull' command

### LINKED ISSUE #27: 'pull' without an argument has to pull changes for current branch
`git elegant pull` always requires a branch as the argument. But the branch has to be optional:
- if not provided, pull changes of the current branch
- if provided, switch to provided branch and pull changes

### PR DESCRIPTION
**Pull request by AntonKarabaza**
Closes #27

### CODE REVIEW

File: src/main/git-elegant [hunk 1]
```diff
@@ -90,10 +90,14 @@ __batch() {
 }
 
 pull() {
-    _validate "$1" "Branch name"
-    git checkout "$1"
-    git fetch --tags
-    git pull
+    if [ -z "$1" ]; then
+        git fetch --tags
```
Comment (extsoft) [inline src/main/git-elegant]:
```
git fetch --tags
git pull
``` 
is used twice. Could we reduce it somehow?

**Comment by AntonKarabaza** [comment]
@extsoft Code duplication reduced. Please, review again.

**Comment by extsoft** [comment]
@AntonKarabaza great job! 😏

**Comment by extsoft** [comment]
@rultor merge

**Pull request merged**